client/layout/controller.js

```diff
@@ -53,11 +54,30 @@ class Controller extends Component {
 		const search = this.props.location.search.substring( 1 );
 		const query = parse( search );
 		const page = find( getPages(), { path } );
+		window.wpNavMenuUrlUpdate( page.wpMenu, query );
 		window.wpNavMenuClassChange( page.wpMenu, this.props.location.pathname );
 		return createElement( page.container, { params, path: url, pathMatch: path, query } );
 	}
 }
 
+// Update links in wp-adming menu to persist time related queries
```

Comment:
Typo? :slightly_smiling_face: 
```suggestion
// Update links in wp-admin menu to persist time related queries
```